core/frontend/src/extension/ExtensionHost.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Extensions
+ */
+
+import { IModelApp } from "../IModelApp";
+import type { ToolAdmin } from "../tools/ToolAdmin";
+import type { AccuSnap } from "../AccuSnap";
+import type { NotificationManager } from "../NotificationManager";
+import type { ViewManager } from "../ViewManager";
+import type { ElementLocateManager } from "../ElementLocateManager";
+import type { Localization } from "@itwin/core-common";
+import type { RenderSystem } from "../render/RenderSystem";
+
+/**
+ * Subset of IModelApp exposed to Extensions
+ * @beta
+ */
+export class ExtensionHost {
```

Comment:
we need to rename this, should also probably be alpha

Reply:
Has there been any names that we liked yet?

Reply:
How 'bout just `Extension`?

Reply:
ExtensionHost seems fine to me.